\newglossaryentry{biomass}{
  name={biomass}, 
  text={biomass reaction},
  symbol={biomass},
  description={an objective 
  reaction consisting of important metabolites that the cell 
  needs in order to grow (e.g.\ amino acids, lipids and 
  carbohydrates). Correlated reaction sets: groups of reactions 
  whose \glspl{flux} always change in relation to one another}}
\newglossaryentry{corset}{
  name={correlated reaction set}, 
  text={correlated reaction set},
  symbol={CoSet},
  description={groups of reactions 
  whose \glspl{flux} always change in relation to one another}}
\newglossaryentry{exchangereaction}{
  name={exchange reaction}, 
  description={a reaction that transports metabolites 
  into or out of the system}}
\newglossaryentry{fluxspace}{
  name={feasible flux space}, 
  description={the possible combination of allowable 
  \glspl{flux} given a defined set of constraints which bound 
  reaction \glspl{flux}, for example reversible and irreversible 
  reactions}}
\newacronym
  [description={a linear programming problem formulated to maximize
  the \glspl{flux} through an objective reaction, which the analyst defines, under
  steady-state flux constraints that are derived from the
  stoichiometric matrix of the metabolic network},
  name={flux balance analysis (FBA)}]
  {fba}{FBA}{flux balance analysis}
\newglossaryentry{fluxdistribution}{%
  name={flux distribution}, 
  description={the set of all reaction 
  \glspl{flux} within a metabolic network}}
\newacronym
  [description={a \gls{linearprogramming}-based method which 
  determines the minimum and maximum reaction \glspl{flux} that 
  allow for optimal or near-optimal \gls{flux} through the objective 
  reaction},
  name={flux variability analysis (FVA)},
  symbol={flux variability}]
  {fva}{FVA}{flux variance analysis}
\newacronym
  [description={the combination of proteins or protein 
  components sufficient to carry out an enzymatic reaction and 
  the combination of genes sufficient to express each of 
  the protein components},
  name={gene-protein-reaction relationship (GPR)}]
  {gpr}{GPR}{gene-protein-reaction relationship}
\newglossaryentry{insilico}{%
  name={in silico}, 
  description={in contrast to in vivo or in vitro, 
  the term indicates a computational process in a 
  simulated environment}}
\newglossaryentry{linearprogramming}{%
  name={linear programming}, 
  description={also termed linear optimization, an area 
  of mathematics developed for maximizing a linear 
  combination of variables (e.g.\ $A v_1 + B v_2 + C v_3$), 
  such that the variables are constrained by many linear 
  equalities and inequalities (e.g.\ the constraint $v_1 - v_2 = 0$ 
  implies that \gls{flux} $v_1$ is constrained to be twice \gls{flux} $v_2$)}}
\newglossaryentry{massbalance}{%
  name={mass balance}, 
  description={a requirement that the mass entering 
  the system or any pathway within the system equals 
  the mass exiting the system or pathway; a crucial 
  characteristic of metabolic \glssymbol{reconstruction}s from which 
  functional mathematical models can be derived}}
\newglossaryentry{reconstruction}{%
  name={metabolic network reconstruction},
  symbol={reconstruction},
  description={a manually-curated computational network 
  of the metabolism of an organism with all the \glspl{gpr} 
  assembled from a functionally annotated genome, biochemical 
  data, and literature, that are compiled into a 
  \gls{smatrix}, which serves as the framework 
  for further computational analysis}}
\newglossaryentry{objectiveflux}{%
  name={objective flux},
  description={the \gls{flux} through the \gls{objectivereaction}}}
\newglossaryentry{objectivereaction}{%
  name={objective reaction},
  symbol={objective},
  description={a reaction which sets a demand for 
  particular metabolites in the network. The typical 
  goal in formulating this reaction is to simulate 
  a biological objective, whether it be growth, 
  energy, virulence, or a combination of other factors}}
\newglossaryentry{flux}{%
  name={reaction flux},
  text={flux},
  plural={fluxes},
  description={moles consumed in a reaction per unit time}}
\newglossaryentry{steadystate}{%
  name={steady state},
  description={with respect to \gls{flux}, a key assumption 
  in \gls{fba} that the reaction \glspl{flux} within the system, and 
  therefore the amounts of each metabolite, do not change 
  over time, an assumption often justified by the very short 
  time-scale of metabolic reactions compared to the time 
  necessary for changes in cell phenotype}}
\newglossaryentry{smatrix}{%
  name={stoichiometric matrix (S matrix)},
  text={stoichiometric matrix},
  description={a mathematical formalization of a 
  metabolic \glssymbol{reconstruction}; a matrix in which each 
  element contains the stoichiometric coefficient 
  for a metabolite (row) participating in the 
  corresponding reaction (column)}}