
\chapter[In vivo toxin responses]{ In vivo physiological and transcriptional profiling of Toxins A and B }

\section{From in vitro to in vivo host responses}
The cell cycle is disrupted in HCT8 epithelial cells, but does something a similar
thing happen to epithelial cells in the intestines? To answer this,
we isolated epithelial cells from mice injected with toxins.
There was no significant difference in the proportions of cells in each stage of the
cell cycle when comparing treatment groups (data not shown). This may have been
because most of the epithelial cells in the intestines are no longer growing
and so toxins cannot block cell cycle in any way. The cell cycle is already stopped.
We therefore performed immunohistochemistry for changes in cell growth markers
Ki67 and PCNA at the base of intestinal crypts (the locations where cells are still
dividng and growing). The results were not definitive (data not shown), but did
indicate that their might be changes in cell cycle for the small minority of
epithelial cells that are actively dividing and growing. Better markes techniques
will be required to isolate changes in specific cell 
types \cite{Blanpain:2013tq,Barker:2012ul}. For example,
one could use transgenic Fucci mice, whose cells fluoresce red or green depending
on their stage in the cell cycle \cite{SakaueSawano:2008ul}.
In addition to cell cycle regulators, many other genes were altered
in HCT8 cells, but their relevance in the pathogenesis of disease
could not be assessed in vitro. In this chapter, I present altered gene expression
induced by TcdA and TcdB in a mouse intoxication model. Several novel
responses were identified, and comparisons between in vitro and in vivo
experiments are discussed in the last sections.


\section{Synopsis}

Toxin A (TcdA) and toxin B (TcdB) of \textit{Clostridium difficile} cause gross 
pathologic changes (e.g., inflammation, secretion, and diarrhea) in the infected
host, yet the molecular and cellular pathways leading to observed host responses
are poorly understood. To address this gap, we evaluated the effects of single
doses of TcdA and/or TcdB injected into the ceca of mice and several endpoints
were analyzed, including tissue pathology, neutrophil infiltration, epithelial-layer 
gene expression, chemokine levels, and blood-cell countsâ€”2, 6, and 16h after
injection. In addition to confirming TcdA's gross pathologic effects, we 
found that both TcdA and TcdB resulted in neutrophil infiltration. Bioinformatics 
analyses identified altered expression of genes associated with the metabolism 
of lipids, fatty acids, and detoxification; small GTPase activity; and immune
function and inflammation. Further analysis revealed transient expression of 
several chemokines (e.g., Cxcl1 and Cxcl2). Antibody neutralization of CXCL1
and CXCL2 did not affect TcdA-induced local pathology or neutrophil 
infiltration, but it did decrease the peripheral blood neutrophil count. 
Additionally, low serum levels of CXCL1 and CXCL2 corresponded with greater 
survival. Though TcdA induced more pronounced transcriptional changes than 
TcdB and the upregulated chemokine expression was unique to TcdA, the 
overall transcriptional responses to TcdA and TcdB were strongly correlated, 
supporting differences primarily in timing and potency rather than differences 
in the type of intracellular host response. In addition, the transcriptional 
data revealed novel toxin effects (e.g., altered expression of GTPase-associated 
and metabolic genes) underlying observed physiological responses to 
\textit{C. difficile} toxins.

\begin{figure}[h!]
  \centering
  \includegraphics[width=\textwidth]{ini/Figure1}
  \caption[Study workflow]{
       \textbf{Study workflow}.
  }
  \label{ini:fig1}
\end{figure}


\section{Introduction}

The toxins TcdA and TcdB are two key virulence factors of \textit{C. difficile}, 
an intestinal, opportunistic pathogen responsible for more than 300,000 infections 
in the US per year (2009 data) with several estimates of annual cost between 
\${}433 million and \${}8.2 billion 
\cite{Lucado:2012wl, Ghantoji:2010ku, McGlone:2011fh, Dubberke:2012hk}. 
Clinical manifestations include leukocytosis and diarrhea. The 
importance of TcdA and TcdB is underlined by the facts that strains 
without either toxin colonize but do not cause disease and that 
intoxication causes similar manifestations as infection 
\cite{Lyras:2009jx,Kuehne:2010hv,Libby:1982wm}. TcdA and TcdB are similar 
in size, amino acid sequence, and enzymatic specificity, yet exhibit 
different enzymatic activities and in vivo potencies 
\cite{vonEichelStreiber:1992tg,ChavesOlarte:1997cs,Taylor:1981ud}. 
Furthermore, much remains unknown about common and divergent 
cellular pathways leading to toxin-mediated host responses 
\cite{Jank:2008eq,Carter:2012cm}.

Determining the relative roles of TcdA and TcdB in pathogenesis has 
proven difficult in part because of variable findings within and 
between animal models as well as species-specific responses. Clinically, 
strains lacking TcdA are commonly isolated from infected patients, and 
no TcdA+/TcdB- clinical strain has ever been reported \cite{Drudy:2007fa}. 
Toxin effects in the context of infection have typically been studied 
using animal models in which an antibiotic regimen and subsequent 
disruption of intestinal flora must precede infection with 
\textit{C. difficile} \cite{Small:1968ux,Bartlett:1977wr}. By generating 
mutant strains, Lyras et al. found that TcdB but not TcdA was 
essential for hamster infection, yet Kuehne et al. found, in a 
similar hamster infection model, that either toxin was sufficient 
\cite{Lyras:2009jx,Kuehne:2010hv}. Investigating toxin effects 
more directly, multiple intoxication models have demonstrated 
TcdA to be enterotoxic, while TcdB caused little to no pathology 
\cite{Libby:1982wm,Taylor:1981ud,Lyerly:1985dx}. However, epithelial 
damage in human xenografts in mice is greater with TcdB than TcdA, 
suggesting that many differences in toxin effects may be species-specific 
\cite{Savidge:2003ck}. The ability of either toxin to bind, enter, 
and/or activate intestinal cells may also explain differential 
effects of TcdA and TcdB. The sequences differ most in the C-terminal 
binding domain. TcdB has been shown to be incapable of binding the 
brush border membranes of hamsters, although TcdB has been found to 
further damage bruised ceca, synergize with TcdA, and contribute to 
pathogenesis during infection \cite{Lyras:2009jx,Lyerly:1985dx,Rolfe:1991vx}. 
Multiple receptors for TcdA have been proposed or identified, yet the 
roles of these receptors in different organisms, animal models, and 
cell types are unclear 
\cite{Krivan:1986tq,Tucker:1991uk,Rolfe:1995wl,Pothoulakis:1996ba,Pothoulakis:1996kd,Na:2008eu}. 
TcdB weakly binds various trisaccharides and oligosaccharides, yet 
no functional receptor for TcdB has been identified \cite{ElHawiet:2011ep}. 
It is also possible that differences in intracellular actions of TcdA 
versus TcdB are responsible for differences in the host response. Though 
a similar dose of TcdA or TcdB may result in different gross pathologies, 
it is unclear if entirely different pathways are activated or repressed 
or if the same overall functions are affected to different degrees. We 
previously analyzed the transcriptional response of a human, ileocecal, 
epithelial cell line (HCT8) to TcdA and TcdB and showed that the toxins 
induce very similar transcriptional signatures, yet the effects of TcdB 
occurred earlier \cite{DAuria:2012bd}. In addition, we found altered 
regulation of many genes involved in cell growth and division but no 
overwhelming expression of inflammatory markers or other genes associated 
with physiological changes in vivo. The in vivo effects of these toxins 
have not been investigated by measuring genome-wide responses, and many 
of the links between cellular responses and physiological changes remain 
unknown. We therefore used an in vivo system, intracecal injection of toxin 
into mice, and collected samples to characterize the genome-wide cellular 
responses and gross physiological effects of each toxin over a 16h time course.

It has been difficult to tease apart the aspects of the host response to TcdA 
and TcdB because of the important interactions among the many tissues, cell 
types, and signals involved \cite{Sun:2010kt,Madan:2012kp}. The intestinal 
epithelium, the initial barrier to these toxins, continuously interacts 
with surrounding cells throughout the development and resolution of disease. 
We therefore focused on the transcriptional response of epithelial-layer 
cells to toxin and other toxin-related effects. Given the importance of 
surrounding tissues and with recent evidence of systemic dissemination of 
toxins, we chose cecal injection of toxin, an open system, as opposed to 
closed ileal loop models or ex vivo systems that may restrict toxin to a 
limited area \cite{Steele:2012ft}. Additionally, previous studies have 
focused on separate facets of the host response, typically with only one 
toxin per study 
\cite{Morteau:2002ts,Kelly:1994cu,Kokkotou:2009ik,Ishida:2004ta,Castagliuolo:1998um,Warny:2000ct,Alcantara:2005dt}. 
To address these deficits in the knowledge of this illness, we measured 
genome-wide expression from epithelial-layer cells exposed to TcdA and 
TcdB to simultaneously capture effects of each toxin.

Using this approach, we identify several genes differentially expressed 
after toxin treatment that serve as specific candidates to investigate 
further. Additionally, we employ currently available bioinformatic methods 
and also introduce novel methods to identify groups of regulated genes 
associated with known biological functions. Our measurements were taken 
on several biological levels to link changes in one set of variables 
(e.g., gene expression) to changes in others (e.g., pathology and blood counts). 
These linkages serve as tools to validate previous findings as well as 
identify novel functions affected by TcdA or TcdB. Of the many linkages 
that could be explored, we further investigated chemokine expression and 
the role of two chemokines in the response to TcdA cecal injection with 
respect to changes in pathology, neutrophil recruitment, and survival. 
In addition to the comparison between toxins and identification of differentially 
expressed genes, these associations and concepts serve as a basis for 
further probing the host response to these toxins in the context of 
\textit{C. difficile} infection.

\section{Methods}

\subsection{ Cecal injection }
All procedures involving animals were conducted in accordance with the 
guidelines of the University of Virginia IACUC (Protocol \#3626). Purified 
TcdA and TcdB were generously provided by Dr. David Lyerly at TECHLAB, Inc. 
Mice (male C57BL/6J, 8 w.o. from Jackson Laboratories) were anesthetized 
with ketamine/xylazine in preparation for surgery. A midline laparotomy 
was performed to locate the cecum, and 20 \textmugreek{}g of toxin in 100 \textmugreek{}L of 0.9\% 
normal saline was injected into the distal tip. Incisions were sutured, 
and animals were monitored during recovery. Sham injected animals received 
only 100 \textmugreek{}L of saline. If an animal became moribund (i.e., hunched 
posture, ruffled coat, or little to no movement), they were immediately euthanized. 

\subsection{ Cell culture }
An immortalized, C57BL/6 mouse, cecal epithelial cell line (passage 9) 
was provided from the laboratory of Dr. Eric Houpt and maintained as 
described by Becker et al. \cite{Becker:2010io}. Toxin cytopathicity 
was assessed by measuring changes in cell adherence and morphology 
using a multi-well, continuous, electrical impedance assay (xCelligence; 
ACEA Biosciences). In each well, 20x solutions of TcdA or TcdB (prepared 
in media) were added 34h after seeding 21,000 cells yielding the indicated concentrations.

\subsection{ Blood counts }
Blood was collected using cardiac puncture and complete blood counts were 
measured using a HEMAVET 950FS (Drew Scientific). Serum was analyzed for 
levels of systemic chemokines using MILLIPLEX MAP beads, and the signal 
was measured using a Luminex 100 IS System (UVA Flow Cytometry Core Facility).


\subsection{ Histology }
\phantomsection
\label{ini:methods:histology}
A cross section from the middle of each cecum was dissected and fixed. 
The tissues were paraffin-embedded, sectioned, and stained by the UVA 
histology core. H\&{}E sections were coded and scored by a blinded observer 
using parameters to assess inflammation, luminal exudates, mucosa thickening, 
edema, and epithelial erosions \cite{Pawlowski:2010il}. Each of these five 
parameters was scored between zero and three yielding 
total pathology scores between zero and fifteen. Eosinophils were 
detected in tissue using Congo Red Staining \cite{Albert:2011jy}. 
Tissues for H\&{}E, MPO, and eosinophil staining were fixed in 
Bouin's solution; tissues for other measurements were fixed in 4\% 
paraformaldehyde. Immunohistochemistry was performed by the 
Biorepository and Tissue Research Facility at the University of 
Virginia. Monocytes/macrophages, dendritic cells, and neutrophils 
were separately identified using the markers F4/80 (clone CI:A3-1; 
AbD Serotec), Ly75 (EPR5233; Abcam), and myeloperoxidase (MPO; 
rabbit-anti-MPO; Novus Biologicals), respectively. The presence 
of neutrophils was quantified by averaging the number of positive 
cells associated with epithelial and subepithelial layers in ten 
random fields (40x objective). Monocytes/macrophages and dendritic 
cell staining was scored by analyzing each section for the number 
of positive cells and overall staining intensity.  Samples were 
assigned scores of 1 (few cells/weak staining), 2 (moderate staining), 
or 3 (many cells/intense staining). 

\subsection{ Isolation of cells from cecal tissue }
The remaining two sections of each cecum were opened longitudinally, rinsed 
with Hank's Balanced Salt Solution (HBSS, Gibco), and shaken at 250 rpm for 
30 minutes at 37\textdegree{}C in HBSS containing 50mM EDTA and 1mM DTT 
in order to remove epithelial layer cells. The digested tissue was strained 
with  a 100-\textmugreek{}m cell strainer, and the filtrate was centrifuged 
(1,000$\times$g, 4\textdegree{}C, 10 min). Cells were resuspended in 
red-cell lysis buffer (150 mM NH4Cl, 10mM NaHCO3, 0.1 mM EDTA) and 
centrifuged again. The pelleted cells were used immediately for flow 
cytometry or stored at -80\textdegree{}C until RNA isolation. RNA was 
isolated using the RNeasy mini kit (Qiagen) with on-column DNase 
digestion according to manufacturer's instructions. Protein was collected 
from cell lysate supernatants, which were made using a lysis buffer 
containing 50 mM HEPES, 1\% Triton-X100, and HALT protease inhibitor. 
The lysate was incubated on ice for 30 minutes and centrifuged 
(13,000$\times$g, 4\textdegree{}C, 10 min). Clarified supernatants 
were stored at -80\textdegree{}C.

\subsection{ Flow cytometry }
Epithelial layer cells were isolated from mice 16h after injection 
with TcdA (n=2), TcdB (n=3), or saline control (n=3). The cell preparations 
were stained with markers including CD3Îµ (BV421; BioLegend), cytokeratin 
(PE; Novus Biologicals), CD11b (APC-Cy7; BioLegend), B220 (FITC; BD Bioscience), 
and CD45 (V500; BD Bioscience) to detect different populations.  Samples 
were analyzed on the CyAn ADP analyzer; 50,000 events were collected and 
subsequently analyzed using FlowJo software.  

\subsection{Antibody-mediated neutralization of chemokines}
For each antibody, 100 \textmugreek{}g was administered by intraperitoneal 
injection 16h before sham/toxin injection. Mice received a combination 
of anti-CXCL1 (clone 48415) and anti-CXCL2 (clone 40605) or the relevant 
isotype controls (clones 54447 and 141945). All antibodies were 
purchased from R\&{}D systems and resuspended according to the manufacturer's 
directions. Decreased levels of CXCL1 and CXCL2 in the serum of mice 
receiving neutralizing antibody (compared to isotype controls) were 
indicative that the neutralization was successful.

\subsection{ Microarray procedure }
An Agilent 2100 BioAnalyzer was used to assess RNA integrity. cDNA 
was synthesized, biotin labeled, and hybridized to Affymetrix Mouse 
Genome 430 2.0 GeneChip according to the manufacturer's instructions. 
Arrays were scanned with a GeneChip System 3000 7G (Affymetrix).

\subsection{ Statistical analysis }
Unless otherwise stated, each two-sample test is a two-sided Mann-Whitney U test.

\subsection{ Microarray preprocessing }
\phantomsection
\label{ini:methods:bioinformatics}

Multiple steps are involved in processing the raw data of microarrays
to a matrix of signal intensity values. We first 
introduce the usual steps and then describe the methods we chose
for each step.

\subsubsection{Possible steps in processing microarrays}

Except for probe set summarization, each one of the steps below is
optional. The order of some steps may be alternated, though this is not
typical. 

\begin{enumerate}
  \item \emph{Background correction}: Each array is corrected for background intensity.
  \item \emph{Probe level normalization}: All arrays are normalized to each 
  other based on the probe level data.
  \item \emph{Mismatch correction}: The signal intensity of perfect-match probes 
  is corrected based on the signal from corresponding mismatch probes.
  \item \emph{Probe set summarization}: The probes in each probe set are 
  summarized into one probe set signal intensity
\end{enumerate}

Each step has about four to ten different computational
methods that can be used. Many of the methods have several optional parameters.
Hence, the number of ways to process microarrays is vast.

\subsubsection{Steps taken in processing microarrays}

In the list below are the computatonal methods we chose for the microarray
processing steps described in the last section. 
Instead of presenting the lengthy comparative analysis of the many
sequences of methods that we examined, we provide only a brief description of either 
why a method was chosen or why others were not.

\begin{enumerate}
  \item \emph{RMA Background correction} \cite{Irizarry:2003ge}: 
  gcRMA\cite{Wu:2004wh} was not used since it
  has been shown to cause artificially high gene-gene correlation which
  would affect our gene set enrichment tests \cite{Lim:2007gc}.
  
  \item \emph{Cyclic loess probe level normalization}: Cyclic loess normalization was 
  used over quantile normalization because of quantile normalization's assumption
  that the probes on each microarray have the exact same 
  distribution \cite{Ballman:2004ee}. Invariant
  set normalization caused some microarrays to become outliers, which we observed
  using principal components analysis \cite{Li:2001wk}. A second cyclic loess normalization was applied
  at the probe set level.
  
  \item \emph{Perfect match probes only}: Only perfect match probes were used.
  Although several model-based processing techniques have shown that mismatch
  probes provide useful information, the precise way in which mismatch probes should
  be used to correct for the signal from the corresponding perfect match 
  probes remains unclear \cite{Wang:2007dy}.
  
  \item \emph{Median polish probe set summarization}: Median polish was 
  used as opposed to Tukey's Bi-Weight algorithm to better account for 
  differences in probe affinities \cite{Tukey:1977uy, Affymetrix:2002vh}.
    
\end{enumerate}

\subsection{Determining differentially expressed genes}\label{S:DE}

As with processing raw microarray data, there are many methods and options
for determining the significance of a probe set's change in expression between
two sample groups. We analyzed the results from many available methods, but only
briefly explain some of the findings that led us to the statistical test 
which we chose. Though permutation tests such as significance analysis of
microarrays are advantageous because they are less sensitive to outliers,
we did not use such tests because our low sample numbers did not allow
many permutations to be generated \cite{Tusher:2001kk}. We did not use 
simple fold change cutoffs because
they do not account for variation among replicates. We also considered limma's 
moderated t-statistic \cite{Smyth:2004gh}. We found
that many of the significant probe sets identified by limma had low intensity values but very
small expression differences between sample groups. This was due to the near zero standard 
errors of some probe sets which we believed to occur for mostly technical,
not biological reasons. We therefore used cyberT, which, in practice,
adjusts the standard deviation for a probe set depending on the signal intensity
of that probe set \cite{Baldi:2001ul}.

The next problem is deciding what p-value cutoff to use for determining which
probe sets are ``significant''. To help with this, various ``p-value adjustments'' or
``FDR corrections'' use the p-value distribution to estimate the false discovery
rate (FDR). FDR corrections generate q values that describe the false discovery
rate for a group of genes. For instance, if the tenth ranked gene has a q value of 0.2, then
2 of the top ten genes should be false positives and the other 8 should be true
positives. We chose to use the Benjamini-Hochberg p-value adjustment
instead of other more conservative methods such as the Bonferroni correction which
controls the family-wise error rate \cite{Benjamini:1995ws}. This 
helps conceptually, yet still leaves an arbitrary choice of 
what q-value cutoff to use. We chose a cutoff of q\textless{}0.01 for mostly the practical
reason that the number of significant genes it led to was manageable.

Of the many statistical tests that we analyzed, we found that the number of
differentially expressed genes varied from test to test. However, for each test,
the number of differentially expressed genes for one comaparison relative to
another comparison was similar. For instance,
if statistical test $A$, relative to test $B$, found double the number of differentially expressed genes
for the TcdA versus sham comparison at 2h, then test $A$ also found about double the number of
differentially expressed genes in all other comparisons. Also,
the ranking of the genes was fairly consistent for different statistical tests.
Hence, in our next analysis of gene set enrichment, we sought enrichment
methods which use expression values or test statistics, not those that require 
genes to be set as ``differentially expressed'' or ``not differentially expressed''.

\subsection{Gene set enrichment}

As briefly mentioned in the manuscript, gene set enrichment methods help to 
understand the biological functions associated with changes in gene expression
data. To define gene sets, we used gene function annotations from the Gene 
Ontology and Reactome databases.

Much of the bioinformatics literature has categorized
different enrichment methods by the type of hypothesis tested \cite{Ackermann:2009bw}.

\subsubsection{Competitive}

One possible hypothesis tested in the ``competitive'' gene set enrichment method
CAMERA, developed by Wu \emph{et al.}, is whether the mean t-statistic for the 
genes in a gene set is significantly different than the genes not in the 
gene set \cite{Wu:2012kp}. In other words, it tests whether or not the genes in a gene set are 
more differentially expressed than the genes outside of the gene set. As 
this type of test compares the differential expression of two competing 
groups of genes (genes in the set versus not in the set), it has come to 
be named a ``competitive'' test. CAMERA was written to use the moderated t-statistic 
from limma \cite{Smyth:2004gh}. We wrote a modified version of CAMERA to instead use
the cyberT t statistic, the same statistic used in our differential gene
expression analysis \cite{Baldi:2001ul}.

\subsubsection{Self-contained}

Another important and common null hypothesis is that the genes in the gene set are 
not differentially expressed at all. Since this type of test requires only data 
from genes in the gene set, it has been coined a ``self-contained'' test. 
Our self-contained method tests whether the avearge t-statistic for the genes 
in a gene set is zero.

To describe our self-contained test, we use the 
notations from the CAMERA manuscript \cite{Wu:2012kp}.
Consider a gene-wise statistic, $z_i$ for gene $i$ in
a gene set with $m$ genes. In our self-contained test, this statistic is the 
standard normal deviate which has the
same quantile as the cyberT t-statistic; a similar transformation is also 
performed with CAMERA. Under the null hypothesis that
the average of the gene expression differences between the two sample groups is zero, 
the gene set's mean test statistic, $\bar{z}$, is normally distributed with
$\mu{}=0$ and

\begin{equation}
Var( \bar{z} ) = \frac{1}{m^2} 
                 \left(   
                   \sum\limits_{i=1}^m \sigma{}_i^2 + 
                   \sum\limits_{i<j} \sigma{}_i \sigma{}_j \rho_{ij}
                 \right).
\end{equation}

Since $z_i \sim \mathcal{N}(0,1)$ for all $i$,

\begin{equation}
Var( \bar{z} ) = \frac{1}{m}  + \frac{m-1}{m}\bar{\rho}
\end{equation}

where $\bar{\rho}$ is the average correlation between genes in the gene set. Note for
large $m$, $Var(z) \approx \bar{\rho}$. If $\bar{z_k}$ is the 
mean test statistic for gene set $k$ calculated from the data, then a p-value
is calculated using location of $\bar{z_k}$ in the cumulative distribution 
function of $\bar{z}$. This novel method improves upon another published, parametric gene
set enrichment method, namely PAGE \cite{Kim:2005fw}.


\subsection{Cytopathic effects on a mouse, cecal, epithelial cell line}

\begin{figure}[b!]
\centering
\includegraphics[width=\textwidth]{ini/supplement/cytopathiceffects.png}
\caption{Cytopathic effects of TcdA and TcdB on a mouse epithelial cell line.}
\label{ini:fig:cells}
\end{figure}

The cytopathic effects (e.g. cell rounding) of TcdA and TcdB were confirmed using an
immortalized, mouse, cecal epithelial cell line (Methods). These effects
were quantified by continuously measuring the impedance across the surface
of electrode-embedded wells (ACEA Biosciences). Similar methods and instrumentation
have been used previously in an ultrasensitive assay to detect TcdA and TcdB. \cite{He:2009hg} 
Changes in impedance represent changes in
cell adherence, morphology, and number. Our visual observations of cell rounding
corresponded with changes in impedance. 

\autoref{ini:fig:cells} shows impedance before and after toxin
addition. All readings are normalized to the impedance
at the time the toxin was added. Gray ribbons surrounding the lines represent
the standard deviation (n=2 per concentration, n=3 for control). If ribbons are not visible, 
this is because the replicates are highly similar. An impedance of zero
is the impedance of media before cells were seeded.

The minimum concentration of 
TcdA needed to alter an impedance profile (as shown in \autoref{ini:fig:cells}) is
between 10 and 100 pg/ml (in a volume of 210.5 \textmugreek{}l); the concentration for TcdB is 
approximately 10 fg/ml. 
Hence, TcdB is at least 1,000 times more potent than TcdA
\emph{in vitro} with immortalized epithelial cells from a mouse cecum. Interestingly, our results
clearly show that TcdA more potently induces inflammation and
tissue damage \emph{in vivo}. 
The initial concentration in our cecal injection experiments was 
20\textmugreek{}g/100\textmugreek{}l=200\textmugreek{}g/ml. 
Nevertheless, the purifications of TcdA and TcdB used
in this study affect mouse cells rapidly (e.g. $<$10min for TcdB at 100 ng/ml, 
\autoref{ini:fig:cells}) at concentrations comparable to and even far lower than 
the initial concentrations in our \emph{in vivo} experiments.


\section{Results}

\subsection{Dose response to TcdA cecal injection}

To understand the potency of TcdA in a cecal injection system, a dose response
experiment was performed with histopathology as the measured outcome.
Five histopathology parameters were scored from zero to three, for a total possible score
between zero and fifteen (Methods). Excluding the largest dose (40 \textmugreek{}g),
TcdA dose was positively correlated with the parameters ``Architecture and epithelial erosions'', 
``Congestion and luminal exudates'', and
``Inflammation'' (\autoref{ini:fig:dose}). The total histopathology score was similarly correlated.
The 20 \textmugreek{}g dose led to the highest scores; the scores with
the 40 \textmugreek{}g dose were similar to the scores for the 5 or 10 \textmugreek{}g dose.
A slight correlation between toxin dose and the ``Edema'' and ''Mucosal thickening''
parameters may be observed from \autoref{ini:fig:dose}, yet this possible
correlation is unclear due to Sham mice scoring similar to all other toxin doses.
TcdA thus affects aspects of histopathology beginning at 5 \textmugreek{}g, the lowest
dose tested. Maximal effects occur with 20 \textmugreek{}g, the dose used in all our
other experiments.

\begin{figure}[h!]
\centering
\includegraphics[width=\textwidth]{ini/supplement/doseresponse.pdf}
\caption{Histopathology 6h after cecal injection with TcdA.}
\label{ini:fig:dose}
\end{figure}

\subsection{Survival after cecal injection}

In addition to the dose response experiment, three cecal injection experiments were performed, each on
different days (\autoref{ini:tab:sizes}). 

\begin{table}[h!]
\centering
\begin{adjustwidth}{-1cm}{}
\resizebox{1.12\textwidth}{!}{%
\begin{tabular}{ c | c c c c | c c c c | c c c | c c }
   & \multicolumn{4}{c|}{2 hours} & \multicolumn{4}{c}{6 hours} & 
     \multicolumn{3}{|c|}{16 hours} & TcdA & TcdB \\
  Toxin: & A & B & A+B & Sham & A & B & A+B & Sham & A & B & Sham & Lot \# & Lot \# \\ \hline
  Dose response & --- & --- & --- & --- & 15/16* & --- & --- & 3/3 & --- & --- & --- & 0810123 & --- \\
  Exp. 1 & 3/3 & 5/5 & 3/3 & 3/3 & 3/3 & 3/3 & 1/3 & 3/4 & 3/3 & 2/3 & 3/3 & 0909101 & 0209165 \\
  Exp. 2 & 3/4 & 4/4 & --- & 4/4 & 3/4 & 3/4 & --- & 3/4 & 5/5 & 4/5 & 5/5 & 0909101 & 0209165\\
  Exp. 3 & --- & --- & --- & --- & --- & --- & --- & --- & 2/5 & 3/4 & 3/3 & 0810123 & 0209165\\ \hline
  Totals & 6/7 & 9/9 & 3/3 & 7/7 & 10/11 & 6/7 & 1/3 & 9/11 & 10/13 & 9/12 & 11/11 &  & \\
   & 86\% & 100\% & 100\% & 100\% & 91\% & 86\% & 33\% & 82\% & 77\% & 75\% & 100\% &  & \\
\end{tabular}
}
\end{adjustwidth}
\caption[Survival in cecal injection experiments]
    {\textbf{Survival in cecal injection experiments}
The ``15/16*'' in the table are the mice shown in
\autoref{ini:fig:dose}. Only four of these mice were given 20 \textmugreek{}g, and
only these four are included in the ``Totals''. The one mouse that did not
survive in the dose response experiment was given 5\textmugreek{}g of TcdA.
The TECHLAB\textsuperscript{\textregistered{}} lot numbers 
of the toxins used are given in the last two columns. Mice used for microarrays
and all protein measurements were from experiments 1 and 2. The numbers
give the fraction of survivors in each sample group (\# survivors/total mice).}
\label{ini:tab:sizes}
\end{table}


\subsection{ Physiological results of toxin cecal injection }
TcdA, TcdB, or TcdA and TcdB (TcdA+B) were injected into the cecum 
to study an anatomical site affected during infection (\autoref{ini:fig1}). 
The TcdA dose (20 \textmugreek{}g/animal) and incubation periods (2, 6, and 
16h) were chosen based on our in vitro data and dose-response experiments 
(\autoref{ini:fig:dose}) in order to capture the early effects from a 
single dose of toxin \cite{DAuria:2012bd}. The biological activities 
of TcdA and TcdB were confirmed using an immortalized, mouse, cecal 
epithelial cell line (\autoref{ini:fig:cells}).


Relative to sham controls, TcdA-challenged mice had greater total 
pathology scores (more severe pathology) at 2, 6, and 16h (\autoref{ini:fig2p1}A 
and \autoref{ini:fig2p1}B), with higher scores for all five measured 
parameters at 16h (p$<$0.01), all but mucosa thickening at 6h (p<0.03), 
and all but luminal exudates at 2h (p$<$0.02). In contrast, the average 
total pathology score for TcdB-challenged mice was significantly 
higher than sham mice at only 16h (p$<$0.05). At 2h, TcdA+B led to 
mucosal thickening, inflammation, and edema (p$<$0.04). Mice challenged 
with TcdA experienced diarrhea at intermediate and late time points 
based on visual observations of wet tail and clumped cage bedding. We 
also examined the colons of mice 16h after TcdA, TcdB, or sham challenge 
in order to determine if there are distant effects from the cecal injection. 
Using the same scoring system, no significant differences in 
histopathology scores were noted throughout the colon (data not shown).

\sidecaptionvpos{figure}{t}
\begin{SCfigure}
  \centering
  \includegraphics[width=0.6\textwidth]{ini/Figure2p1}
  \caption[Physiological and gene expression changes post toxin injection]{
       \textbf{Physiological changes post toxin injection.}
       Panels A, B, and C include data combined from four independent experiments 
       with 7, 39, 36, and 12 mice. In total, 10 of 94 mice did not survive until 
       the experimental end point: one TcdA-treated mouse did not survive to 2h, 
       six mice did not survive to 6h (1 TcdA, 2 TcdA+B, 1 TcdB, 2 Sham), and 
       six mice did not survive to 16h (3 TcdB, 3 TcdA; \autoref{ini:tab:sizes}). 
       The data points displayed in the figure were used for each statistical 
       test. The horizontal lines above the bar charts which connect two sample 
       groups indicate a two-sample statistical tests. The p-values for these 
       tests are indicated beside the lines. \textbf{(A)} Representative 
       examples of H\&{}E-stained cecal tissue sections from the eleven 
       indicated sample groups. \textbf{(B)} Total histopathology score 
       (see \ref{ini:methods:histology}) from cecal-tissue sections. Except 
       for the two mice injected with TcdA+B (two mice not used for microarrays), 
       histopathology scores were not measured for mice that did not survive. 
       Since two of three mice injected with TcdA+B did not survive to six 
       hours in our first experiment, we dedicated more mice for TcdA and 
       TcdB at 16h so that no samples were obtained for TcdA+B at 16h. All 
       subsequent experiments also excluded the 16h time point for injection 
       of TcdA+B. \textbf{(C)} The number of cells within the mucosa and 
       immediate submucosa which were positive for MPO after immunohistochemical 
       staining. *p=0.055 by the two-sided t test.      
  }
  \label{ini:fig2p1}
\end{SCfigure}


The complete blood counts and infiltration of immune cells are also altered 
by TcdA and TcdB. In blood drawn by cardiac puncture 16h after injection 
of TcdB, there was an increased concentration of several cell types 
(\autoref{ini:fig:cbcs}). At 16h, TcdA slightly increased the concentration 
of monocytes (p$<$0.1), but decreased the concentrations of lymphocytes 
and platelets (p$<$0.05). The increased systemic concentration of 
monocytes after TcdA challenge is reflected in their infiltration into 
the cecal submucosa 6 and 16h after injection (based on immunohistochemistry 
staining using F4/80, \autoref{ini:tab:ihc}). In addition, increases in 
dendritic cells were also evident in the submucosa 6h and 16h after TcdA 
challenge. Relative to sham, TcdA and TcdB increased neutrophil 
infiltration at 16h (p$\leq$0.02, using MPO staining), and at 6h, 
neutrophil infiltration in four of the six animals challenged with TcdA 
was greater than neutrophil infiltration in any sham mouse (\autoref{ini:fig2p1}C).


\begin{figure}[ht]
\centering
\begin{adjustwidth}{-1cm}{}
\includegraphics{ini/supplement/cbcs.pdf}
\end{adjustwidth}
\caption[The concentration of circulating
blood cells was altered during intoxication]
      {\textbf{The concentration of circulating
           blood cells is altered during intoxication}
           Blood was drawn by cardiac puncture immediately after mice were euthanized.
Only measurements for mice which survived to the end point are represented
in the figure. Sufficient blood to measure complete blood counts was not 
capable of being drawn from every mouse. The mean corpuscular volume (MCV), 
mean corpuscular hemoglobin (MCH), MCH concentration (MCHC), red blood
cell distribution width (RDW), and mean platelet volume (MPV) were
similar across all three sample groups and are not shown.
           }
\label{ini:fig:cbcs}
\end{figure}

\begin{table}[ht]
\centering
\begin{tabular}{ r c c c c }
  Cell type: & Macrophages & Dendritic Cells & Eosinophils &  \\
  Marker: & F4/80 & Ly75 & Congo Red & \emph{n}\\ \hline
  Sham, 6h: & + & + & + & 4 \\
  TcdA, 6h: & +++ & ++ & + & 8 (4 for F4/80) \\
  Sham, 16h: & + & + & + & 4 \\
  TcdA, 16h: & +++ & +++ & + & 7 \\
\end{tabular}
\caption[Infiltration of immune cells 6h and 16h after cecal injection]
          {\textbf{Infiltration of immune cells 6h and 16h after cecal injection}
          To determine if monocytes/macrophages, dendritic cells, and eosinophils
infiltrate the epithelial layer after cecal injection of toxin, we used
immunohistochemistry markers (Methods). Cecal tissue sections were scored 
by analyzing each section for the number of positive cells and overall
staining intensity.  Samples were assigned scores of + (few cells/weak staining), 
++ (moderate staining), or +++ (increased cell/intense staining).
          }
\label{ini:tab:ihc}
\end{table}


\subsection{ Host transcription altered by toxin injection }
We evaluated gene expression changes in the epithelial layer to 
characterize the host-cell responses to TcdA and TcdB. To determine 
the proportions of cell types within our epithelial-layer isolation, 
we used flow cytometry to analyze cells from mice 16h after TcdA, TcdB, 
and sham challenge (\autoref{ini:fig:flow}). The percentage of epithelial 
cells (cytokeratin+) was similar across all experimental conditions, 
averaging 85\%. Approximately half of the remaining non-epithelial 
fraction in each experimental condition was leukocytes (CD45+). However, 
the number of leukocytes was slightly greater for TcdA-challenged mice 
over sham controls (p=0.06). Within the leukocyte fraction, there was 
no significant difference between experimental conditions in the 
percentage of CD3+ cells (average of 40\%, T cell marker) or CD11b+ 
cells (average of 59\%, marker for myeloid lineage). However, the 
percentage of leukocytes positive for B220, a common B cell marker, 
was greater in TcdB samples than sham samples (60\% versus 34\%, 
p$<$0.01). The remaining non-epithelial fraction, an average of 7.5\% 
of all cells, was not characterized by the markers used. The same 
epithelial-layer isolation procedure was used for all gene expression 
measurements.

\begin{figure}[h!]
  \centering
    \includegraphics[height=6in]{ini/supplement/flow.pdf}
  \caption[Cell type proportions as determined by flow cytometry]
  {\textbf{Cell type proportions as determined by flow cytometry}
  The cell type proportions within the epithelial-layer cell isolation
were determined by flow cytometry. Two
separate flow cytometry panels were used. For each sample, 50,000
events, or particles, were captured. 
For both panels, all particles were initially gated using pulse 
width and forward scatter-area to select for single 
cells, or singlets.  Events in this gate were then separated 
by forward scatter and side scatter to identify epithelial 
and non-epithelial fractions.  Epithelial cells were then 
validated using cytokeratin positivity in the first panel.  
The non-epithelial fractions from both panels were further characterized
with CD45, a leukocyte marker. Finally, the leukocytes were further characterized
using CD3 (a T cell marker), CD11b (a myeloid lineage marker), and 
B220 (a primarily B cell marker).}
  \label{ini:fig:flow}
\end{figure}


For both toxins, a small set of genes is affected at 2h and, at 6h 
and 16h, hundreds or thousands are differentially expressed. A more 
pronounced TcdA transcriptional response, compared to TcdB, is 
consistent with TcdA's greater pathophysiological effects in vivo 
(\autoref{ini:fig2p1}B). However, many of the expression changes 
induced by TcdA and TcdB are similar. Over 50\% and 90\% of the 
genes that are differentially expressed after TcdB treatment, at 
6h and 16h respectively, are also differentially expressed after 
TcdA treatment (\autoref{ini:fig2p2}A). Comparing challenge with 
individual toxins versus TcdA+B challenge, the transcriptional 
response induced by TcdA+B is very similar to that induced by TcdA 
alone. For instance, at 2h, 12 of the 20 genes with a significant 
change in expression after TcdA challenge were also differentially 
expressed after TcdA+B challenge. Although the degree of pathology 
and magnitude of transcriptional changes are significantly 
different between TcdA and TcdB, the overall transcriptional 
responses to TcdA and TcdB are highly correlated (\autoref{ini:fig2p2}B). 
Hence, in general, gene expression that is affected by TcdA is also 
affected by TcdB but to a lesser extent, suggesting broadly similar 
in vivo cellular responses to TcdA and TcdB.

\begin{SCfigure}
  \centering
  \includegraphics[width=0.6\textwidth]{ini/Figure2p2}
  \caption[Gene expression changes post toxin injection]{
       \textbf{Gene expression changes post toxin injection.}.
       \textbf{(D)} Venn diagrams show the overlap of which microarray 
       probe sets are differentially expressed (comparing toxin-challenged 
       mice versus Sham-challenged mice using a cutoff of q$<$0.01, see 
       \ref{ini:methods:bioinformatics}). All microarray probes are 
       annotated into 45,501 probe sets, each of which represents 
       the expression of one gene or multiple similarly related 
       genes. Since only one microarray was used for TcdA+B at 6h, 
       statistical tests could not be used to determine differentially 
       expressed genes for that sample group. \textbf{(E)} All probe 
       sets which were differentially expressed for at least one 
       time point were included in the heat map. The Pearson correlation 
       coefficients below the heat map are generated by comparing the 
       log fold changes between each sample group. The dendrogram above 
       the heatmap is a hierarchical clustering of the sample groups, 
       using the correlation coefficients as the distance metric.    
  }
  \label{ini:fig2p2}
\end{SCfigure}

For an initial perspective of which genes are affected, we present 
genes differentially expressed 2h post toxin challenge 
(\autoref{ini:table1}). These include several transcription factors 
(Atf3, Egr1, and Jun) and an mRNA binding protein (Zfp36). Consistent 
with the observed pathology, several of the affected genes at 2h are 
related to the regulation of inflammation (C3, Cxcl1, Cxcl10, Dusp1, 
Egr1, etc.). Increased expression of Sprr1a and Atf3, markers of 
neuronal damage, is interesting with respect to previous findings 
implicating involvement of the enteric nervous system in the host 
response \cite{Starkey:2009kn,Linhoff:2009dm}. Rhob, upregulated in 
our and others' previous in vitro studies, is also upregulated in 
these experiments \cite{DAuria:2012bd,Gerhard:2005dg}. Manually scanning 
large lists of differentially expressed genes provides novel and 
interesting findings, yet is impractical when the list includes 
thousands of genes as is the case at 6 and 16h. This manual approach 
also overlooks groups of genes that are only slightly regulated, 
but in a coordinated fashion. Therefore, we performed bioinformatics 
analyses to identify other potentially important yet not readily 
apparent associations that could reflect important functional relationships.

\begin{table}[h!]
\begin{center}
\begin{tabular}{ l c c c | c c c | c c }
Time & \multicolumn{3}{c}{2 hours} & \multicolumn{3}{c}{6 hours} & \multicolumn{2}{c}{16 hours}\\
Toxin & A & B & A+B & A & B & A+B & A & B \\
\multicolumn{9}{l}{\textit{Differentially expressed genes after TcdA challenge}} \\ \hline
\textit{Dusp1} & 4.8 & 1.3 & 4.4 & 6.4 & -1.0 & 10.1 & 1.9 & 1.1 \\
\textit{Rhob}  & 4.2 & 1.2 & 3.8 & 4.0 & 1.5 & 4.8  & 2.2 & 1.3 \\
\textit{Atf3}  & 4.0 & 1.4 & 3.5 & 2.7 & -1.2 & 4.0 & 1.2 & 2.0 \\
\textit{Sprr1a} & 3.4 & 1.5 & 3.7 & 4.9 & -1.1 & 8.8 & -1.6 & -1.4 \\
\textit{C3}  & 2.8 & 1.6 & 1.2 & 2.2 & 1.6 & 3.2 & 5.7 & 1.9 \\
\textit{Areg} & 2.6 & -1.1 & 2.5 & 12.8 & 2.4 & 15.5 & 14.9 & 5.1 \\
\textit{Cxcl10} & 2.6 & 2.0 & 6.8 & 11.3 & 1.3 & 13.3 & 4.7 & 2.2 \\
\textit{Insig1} & 2.5 & 1.5 & 1.8 & -1.9 & -1.1 & -1.4 & -2.3 & -1.5 \\
\textit{Errfi1} & 2.5 & 1.1 & 2.2 & 3.4 & -1.0 & 5.5 & -1.2 & -1.4 \\
\textit{Egr1} & 2.3 & -1.1 & 1.5 & 2.5 & 1.9 & 5.0 & 3.7 & 1.4 \\
\textit{Zfp36} & 2.2 & 1.1 & 2.0 & 2.9 & 1.8 & 3.4 & 2.2 & 1.2 \\
\textit{Hmgcs1} & 1.8 & 1.2 & 1.6 & -1.2 & 1.0 & -1.1 & 2.4 & 1.5 \\
\textit{Jun} & 1.8 & 1.3 & 1.6 & 1.4 & 1.2 & 1.6 & -1.9 & -1.1 \\
\textit{Gm11545} & 1.6 & 1.1 & 1.7 & 7.3 & 1.9 & 8.2 & 5.6 & 3.8 \\
\textit{1700006J14Rik} & -1.8 & -1.3 & -1.2 & -1.3 & 1.4 & -1.1 & -1.6 & -1.1 \\
\textit{H3f3b} & -2.0 & -1.5 & -1.3 & 1.1 & 2.0 & 1.4 & 1.8 & 1.4 \\
\textit{Lrrtm1} & -2.3 & -2.0 & -2.2 & -2.3 & -1.5 & -2.5 & -1.9 & -1.9 \\
\textit{Slc8a1} & -2.3 & -1.3 & -1.7 & -3.4 & 1.4 & -2.7 & -2.9 & 1.0 \\
\multicolumn{9}{l}{\textit{Differentially expressed genes after TcdB challenge}} \\ \hline
\textit{Cxcl1} & 4.4 & 2.2 & 6.5 & 9.7 & 1.1 & 8.1 & 3.3 & 1.6 \\
\textit{Mtch2} & -1.5 & -2.0 & -1.7 & -1.1 & 2.0 & -1.6 & -1.5 & -1.2 \\
\textit{Slc20a1} & -1.9 & -2.1 & -2.5 & -2.5 & 1.7 & -2.7 & -4.4 & -1.5 \\
\end{tabular}
\caption[Genes with significantly altered expression 2h after TcdA or TcdB injection]{
 \textbf{Genes with significantly altered expression 2h after TcdA and TcdB injection}
Average fold changes relative to sham are shown (values of -1.1 and +1.1 
imply a 10\% decrease and increase, respectively, in gene expression). 
The cutoff for determining a differentially expressed gene is q$<$0.01 
(see \ref{ini:methods:bioinformatics}). }
\label{ini:table1}
\end{center}
\end{table}


Many statistical tools, termed ``enrichment methods'', can be used 
to determine if the transcription of predefined sets of genes is 
significantly altered or ``enriched''; this approach allows for the 
characterization of cellular processes (instead of individual genes) 
that are affected. Given our experimental design, we carefully 
chose two enrichment methods. In our implementation of a ``competitive'' 
enrichment method named CAMERA, which was developed by Wu et al., we test 
whether the genes in a set are more differentially expressed than those 
outside the set \cite{Wu:2012kp}. We also developed a ``self-contained'' 
test, inspired by CAMERA, to determine if the average change in gene 
expression within each gene set is different than zero. Whereas the 
``competitive'' hypothesis may find a gene set (with several differentially 
expressed genes) to be insignificant because many genes outside the 
set are also differentially expressed, the self-contained hypothesis 
would find the same set to be significant. The self-contained test 
identified enriched functions for all samples; for TcdA and TcdB 
samples at 2h, which have few differentially expressed genes, multiple 
gene sets are enriched (\autoref{ini:table2}). Using the competitive 
test for TcdA samples, no functions are enriched at 6h (q$<$0.2) but 
several are enriched at 16h (\autoref{ini:table3}). The genes within 
these groups are presented below.

\begin{table}[h!]
\begin{center}
\begin{tabular}{ l c c c }
TcdA & $-log_{10}(p)$ & q & Database \\ \hline
Interleukin-1-mediated signaling pathway & 5.3 & 0.003 & BP \\
Cellular response to hydrogen peroxide & 5.1 & 0.003 & BP \\
Positive regulation of fatty acid biosynthetic process & 4.0 & 0.026 & BP \\
Cholesterol metabolic process & 3.7 & 0.042 & BP \\
Hormone activity & 4.2 & 0.021 & MF \\
Innate immunity signaling & 3.9 & 0.047 & Reactome \\
 & & & \\
TcdB & & & \\ \hline
Negative regulation of the Notch signaling pathway & 5.7 & 0.002 & BP \\
Nuclear envelope lumen & 3.6 & 0.045 & CC \\
Genes involved in apoptotic cleavage of cellular proteins & 4.0 & 0.037 & Reactome \\
Membrane trafficking & 3.6 & 0.043 & Reactome \\
\end{tabular}
\caption[Biological functions and gene sets associated with gene 
         expression changes 2h after TcdA or TcdB injection]{
 \textbf{Biological functions and gene sets associated with 
 gene expression changes 2h after TcdA or TcdB injection}
Our self-contained enrichment test was run on multiple databases 
separately, and gene sets with q<0.05 are shown. The logarithms 
of the p-values from the enrichment test are shown. The Gene 
Ontology database is separated into three ontologies: molecular 
functions (MF), biological processes (BP), and cellular components (CC). 
Mouse genes were mapped to human orthologs so that the Reactome 
database could be used.}
\label{ini:table2}
\end{center}
\end{table}


\begin{table}[h!]
\begin{center}
\begin{tabular}{ l c c c }
TcdA & $-log_{10}(p)$ & q \\ \hline
Cell surface binding & 3.5 & 0.102 \\
Rho GTPase binding & 3.3 & 0.102 \\
GTPase activity & 3.1 & 0.107 \\
GTP binding & 2.9 & 0.107 \\
Protein N-terminus binding & 2.5 & 0.205 \\
Glutathione transferase activity & 2.3 & 0.205 \\
Steroid binding & 2.3 & 0.205 \\
Carboxylase activity & 2.2 & 0.205 \\
RNA polymerase II core promoter sequence-specific DNA binding & 2.2 & 0.205 \\
Protein complex binding & 2.1 & 0.205 \\
Heme binding & 2.1 & 0.205 \\
Thiolester hydrolase activity & 2.1 & 0.205 \\
Fibronectin & 2.1 & 0.205 \\
Cholesterol binding & 2.1 & 0.205 \\
Histone deacetylase activity & 2.1 & 0.205 \\
Triglyceride lipase activity & 2.0 & 0.210 \\
Selenium binding & 2.0 & 0.210 \\
Aromatase activity & 2.0 & 0.210 \\
Beta-tubulin binding & 1.9 & 0.210 \\
Actin binding & 1.9 & 0.210 \\
\end{tabular}
\caption[Molecular functions associated with gene expression changes 16h after TcdA injection]{
 \textbf{Molecular functions associated with gene expression changes 16h after TcdA injection.}
The top 20 competitively enriched gene sets from the molecular 
function ontology of the Gene Ontology database are shown.}
\label{ini:table3}
\end{center}
\end{table}



One of the most striking upregulated sets of genes includes those 
encoding proteins which bind to GTP or GTPases (\autoref{ini:table3}). 
These expression changes are most evident for TcdA at 16h, though a 
similar pattern is observed at earlier time points and with TcdB 
(\autoref{ini:fig3}A). The expression of interferon-inducible GTPase 
genes is increasingly affected from 2h to 16h with either toxin. To a 
lesser extent, the expression of several small GTPase genes, not 
directly tied to interferons or immune function, is also altered. These 
small GTPases include members of several subfamilies from the Ras protein 
superfamily. Genes encoding proteins which interact or bind with Rho 
family proteins were also upregulated. Hence, in addition to the 
toxins' glucosylation of small GTPases, the in vivo transcription of 
many GTP and GTPase binding proteins with a wide range of functions 
is clearly altered in response to TcdA and TcdB.

\begin{figure}[h!]
  \centering
  \includegraphics[width=\textwidth]{ini/Figure3}
  \caption[Biological functions associated with gene-expression changes]{
       \textbf{Biological functions associated with gene-expression changes.}
       The expression data was generated from the mice indicated 
       in \autoref{ini:fig2p1}B. \textbf{(A)} The fold changes of 
       differentially expressed GTPase and GTPase-binding genes. 
       Only genes with greater than a two-fold change in expression 
       are shown. \textbf{(B)} Similar to (A), but instead showing 
       genes associated with metabolic functions associated with gene 
       expression changes. \textbf{(C)} Expression changes and clustering 
       of genes annotated as being associated with immune regulation 
       or inflammation. Genes were clustered based on expression 
       changes 6 and 16h after TcdA injection. In the scatterplots, 
       black circles indicate genes with low expression changes; 
       these genes are not included in the line plots.
  }
  \label{ini:fig3}
\end{figure}


We also found an abundance of differentially expressed genes 
associated with cell metabolism (\autoref{ini:table2} and 
\autoref{ini:table3}). More specifically, many enzymes involved 
in fatty acid breakdown and beta-oxidation were downregulated. 
Four other classes of enzymes were also downregulated: cytochrome 
P450 enzymes, glutathione S-transferases, carboxylesterases, and 
sulfotransferases (\autoref{ini:fig3}B). These genes span several 
metabolic pathways, yet there is a strong commonality in their 
substrates, most of which include lipid and fatty acids or related 
compounds; xenobiotics; or both. In addition to the conspicuous 
toxin-induced pathology and inflammation, recognition of the altered 
expression of detoxification enzymes and fatty acid metabolic 
enzymes introduces an unexplored aspect of the host response to 
TcdA and TcdB.

\begin{figure}
  \centering
    \includegraphics[height=7.4in]{ini/supplement/inflam.pdf}
  \caption[Expression changes of inflammation associated
           and immune regulatory genes]{\textbf{Expression changes of inflammation associated
           and immune regulatory genes}
           This figure extends \autoref{ini:fig3}C to also
show TcdB-induced and TcdA-induced expression changes on separate, side-by-side
plots.}
  \label{ini:fig:inflam}
\end{figure}

Inflammation is a clear pathophysiological manifestation of toxin 
injection, and many inflammation-associated genes are differentially 
expressed (\autoref{ini:table1}, \autoref{ini:fig3}C). However, at 
6 and 16h, genes associated with inflammation are not expressed to 
a greater extent than genes associated with several other 
functions (see \autoref{ini:table3}). Competitive enrichment 
tests did find ``inflammatory response'' and ``chemotaxis'' among the top 
six enriched functions for TcdA at 6h, but the enrichment of these 
groups is not significant (q=0.32). Given the importance of 
inflammation in our physiological measurements at 6 and 16h, yet no 
remarkable regulation of only inflammation-associated genes at 
these times, we further investigated the expression of genes known 
to be linked with inflammation and related physiological effects.

To identify temporal expression patterns, we clustered genes 
associated with immune regulation and inflammation according to 
their change in expression over time (\autoref{ini:fig3}C). 
Several of these genes are upregulated at 6h and still at 16h, which 
may represent transcription that perpetuates the inflammatory response 
or has anti-inflammatory effects. For TcdA, expression of five 
chemokine genes (Cxcl1, Cxcl2, Cxcl3, Cxcl10, and Ccl3) is strongly 
upregulated at 6h (coinciding with increased neutrophil infiltration) 
but subsides by 16h. The gene expression of several of these chemokines 
also correlates with protein expression (r=0.67, TcdA at 6h, 
\autoref{ini:fig:geneprotein}). Though the 6h peak in chemokine 
gene expression does not occur with TcdB, TcdA-induced and TcdB-induced 
gene expression changes are correlated for all other inflammatory genes 
(\autoref{ini:fig:inflam}). Hence, except for the aforementioned 
chemokines, TcdA and TcdB similarly regulate inflammation-associated 
and immune-regulatory genes, although TcdA-induced changes are on average 
two and three times greater at 6 and 16h, respectively.




\subsection{Comparisons of gene and protein expression of cytokines}

To test the common assumption that gene expression correlates
with protein expression, we compared the microarray data for chemokines to the
intracellular protein expression. After mapping
microarray probes to the appropriate Ensembl gene IDs, we found that TcdA-induced 
changes in gene expression correlated to changes in protein 
expression (r=0.67, \autoref{ini:fig:geneprotein}A). 

\begin{figure}
\centering
\includegraphics{ini/supplement/cytokines.pdf}
\caption[Cytokine gene and protein expression]
         {\textbf{Cytokine gene and protein expression}
         Our data for Ccl4 is not shown
in \autoref{ini:fig:geneprotein}A because the protein concentration measurements
were at or below the detection limit of our assay. The Ccl4 concentration from
three mice treated with toxin A were 0.8, 0.8, and 4.8 pg/ml; the concentrations
from three sham mice were 0.8, 0.8, and 14.72 pg/ml. Cell 
lysates were obtained as described in the Methods of the Manuscript.
Cytokine levels in serum and lysates were measured by using
MILLIPLEX\textsuperscript{\textregistered{}}
MAP beads and the signal was measured using a Luminex 100 IS System.}
\label{ini:fig:geneprotein}
\end{figure}

\autoref{ini:fig:geneprotein}B compares our measurements of changes in the gene expression 
of cecal epithelial cells (Toxin A+B versus Sham at 6h; cecal injection) 
to protein concentration changes 
in mouse colonic tissue lysates as measured by
Hirota \emph{et al.} (Toxin A+B versus Sham at 4h; intrarectal instillation) \cite{Hirota:2012gx}. 
\autoref{ini:fig:geneprotein}C compares cecal epithelial layer protein expression (Toxin A
versus Sham at 6h) to the same Hirota \emph{et al.} data.
Overall, the changes in the protein concentrations of colonic tissue were greater
than measurements of gene or protein expression from cecal cells. 

In addition to differences in the location of the intestine and 
differences in the biological molecules being measured, the 
Hirota \emph{et al.} samples were obtained from a different source, colonic tissue.
The colonic tissue presumably includes proteins from
intracellular and extracellular proteins, while many of our measurements included
only intracellular protein or mRNA. Nevertheless, \autoref{ini:fig:geneprotein}B
and \autoref{ini:fig:geneprotein}C show correlation between the colonic tissue
data and cecal epithelial layer gene expression (r=0.5) and protein
expression (r=0.36). Additionally, our measurements of changes in blood serum
cytokines after 6h of TcdA injection correlated best with the data from 
Hirota \emph{et al.} (\autoref{ini:fig:geneprotein}D).

\subsection{CXCL1 and CXCL2 neutralization alters the host response to 
            TcdA cecal injection}
To investigate the role of these acutely expressed chemokine genes 
in response to TcdA challenge, we administered neutralizing antibodies 
against CXCL2 and the closely related CXCL1. In addition to the high 
expression of Cxcl2, we also chose Cxcl1 because it is another 
important primary neutrophil chemoattractant. Anti-CXCL1 and anti-CXCL2 
(or corresponding isotypes) were administered by intraperitoneal 
injection (100 \textmugreek{}g/antibody/animal) 16h prior to TcdA cecal 
injection. TcdA-induced increases in the serum levels of CXCL1 and 
CXCL2 is significantly reduced in mice pretreated with anti-CXCL1 
and anti-CXCL2, demonstrating that systemic levels were effectively 
neutralized compared to isotype controls (\autoref{ini:fig4}A; 
p$<$0.01 for CXCL1, p$<$0.02 for CXCL2). To test if neutralization 
of CXCL1 and CXCL2 alters expression of Cxcl1 and Cxcl2, we 
isolated mRNA from epithelial-layer cells. We found that neutralization 
does not eliminate the 6h-peak in Cxcl1 and Cxcl2 expression caused by 
TcdA (\autoref{ini:fig:qrtpcr}). In addition, pathology and neutrophil 
infiltration in the cecum is not affected by administration of 
anti-CXCL1 and anti-CXCL2 (\autoref{ini:fig4}C, \autoref{ini:fig:neutralization}). 
However, 16h after TcdA injection, systemic neutrophil levels are 
reduced by neutralization (\autoref{ini:fig4}B). A larger percentage 
of mice survived after CXCL1 and CXCL2 neutralization, though the 
experiment was not designed to assess survival and more samples would 
be necessary to determine statistical significance (\autoref{ini:fig4}D). 
However, higher sera levels of CXCL1 and CXCL2 correlate with a moribund 
state and administration of anti-CXCL1 and anti-CXCL2 reduces those 
chemokine elevations.

\begin{figure}
  \centering
  \includegraphics[width=\textwidth]{ini/Figure4}
  \caption[Antibody neutralization of CXCL1 and CXCL2]{
       \textbf{Antibody neutralization of CXCL1 and CXCL2.}
        In each panel, the four sample groups are defined by two binary 
        factors: (1) TcdA injection or sham injection and (2) pretreatment 
        with isotype antibodies or anti-CXCL1 and anti-CXCL2 antibodies. 
        The data in all panels are combined from two independent 
        experiments, one with 24 mice and another with 14 mice 
        (\autoref{ini:tab:absizes}). Missing values in panels A and 
        B are due to the limited volume of blood that could be 
        drawn from some mice. The data points displayed in the figure 
        were used for each statistical test. Statistical tests are 
        indicated with horizontal lines as described in the caption 
        to \autoref{ini:fig2p1}. \textbf{(A)} Concentration of CXCL1 
        and CXCL2 in the sera of mice 6h after cecal injection of 
        TcdA. \textbf{(B)} Concentration of neutrophils in blood 
        obtained by cardiac puncture. *p=0.057 by the Mann-Whitney 
        U test; using this nonparametric, two-sided test with three 
        samples in one group and four in the other, the minimum 
        possible p-value is 0.057. p<0.02 by the two-sided t test. 
        \textbf{(C)} Total histopathology score 
        (see \ref{ini:methods:histology}) from cecal tissue 
        sections. \textbf{(D)} Survival of mice after cecal 
        injection. Mice were monitored so that moribund mice 
        were sacrificed and are counted as having not survived (nonsurvivors). 
  }
  \label{ini:fig4}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics{ini/supplement/qrtpcr.pdf}
\caption[\emph{Cxcl1} and \emph{Cxcl2} expression after Cxcl1 \& Cxcl2 neutralization]
{ \textbf{\emph{Cxcl1} and \emph{Cxcl2} expression after Cxcl1 \& Cxcl2 neutralization}
Aside from any possible feedback and regulatory mechanisms, i\@.p\@. administration of anti-Cxcl1 and
anti-Cxcl2 were predicted to neutralize extracellular proteins and not directly affect 
\emph{Cxcl1} and \emph{Cxcl2} expression. Using qRT-PCR to measure mRNA
in epithelial-layer cells, we found that antibody neutralization did not block the transient
increase in \emph{Cxcl1} and \emph{Cxcl2} expression that we previously 
measured by microarray. \emph{Actb} and \emph{Hprt}
were used as ``housekeeping'' genes.
The quantity of each transcript was calculated as $2^{-\Delta \Delta Ct}$ where
$\Delta \Delta Ct$ is the difference in cycle threshold between the transcript and the 
geometric mean of the housekeeping genes).
 }
\label{ini:fig:qrtpcr}
\end{figure}

\begin{figure}[ht]
\centering
\includegraphics{ini/supplement/neutralization.pdf}
\caption[Neutrophil infiltration 6h after TcdA cecal injection 
           of mice pretreated with neutralizing antibodies]
           {\textbf{Neutrophil infiltration 6h after TcdA cecal injection 
           of mice pretreated with neutralizing antibodies}
           We hypothesized that neutralization of Cxcl1 and Cxcl2
would inhibit neutrophil infiltration in the cecum. We found that 
our neutralization did not affect the amount of infiltration after injection
of TcdA (\autoref{ini:fig:neutralization}). The high variability for these 
limited samples, however,
makes it difficult to claim that isotype, anti-Cxcl1, and anti-Cxcl2 antibodies
do or do not have an effect on neutrophil infiltration 6h after cecal injection---at 
least with the method in which we administered the antibodies.
           }
\label{ini:fig:neutralization}
\end{figure}


\subsection{\emph{In vivo} transcriptome response \emph{versus
            in vitro} transcriptome and proteome responses}

\begin{figure}[b!]
\centering
\includegraphics{ini/supplement/correlations.pdf}
\caption{Correlations between \emph{in vitro} and \emph{in vivo} responses to toxins}
\label{ini:fig:correlations}
\end{figure}

In previous studies with epithelial cell lines, the response 
to TcdA and/or TcdB has been analyzed with transcriptomic and 
proteomic techniques. In order to merge the data in this present study
to our previous data from a human ileocecal epithelial
cell line (HCT8 cells) treated with TcdA or TcdB, we mapped orthologous
mouse and human genes \cite{DAuria:2012bd}. These genes were further mapped
to human proteins in order to be compared to a recent proteomics study
by Zeiser \emph{et al.} who treated Caco-2 cells with TcdA \cite{Zeiser:2013cu}.

The fold changes of sample groups relative
to control groups were compared using nonparametric correlation
coefficients (\autoref{ini:fig:correlations}). The values in cells of the figure
are the correlation coefficients squared (the square of the Pearson
correlation coefficient is the coefficient of determination, or $R^2$ value).
The colors of the cells correspond to the correlation coefficients. Negative
correlations are blue; positive correlations are red.
One-to-one orthologs were used to compare the two
transcriptional data sets (19,643 genes). The proteomics data
set was merged separately since data is available for many fewer
proteins than genes (4,090 gene-protein pairs).

As seen above, there is little correlation between different studies
and experimental systems. In \autoref{ini:fig:commons}, we show
13 of the most differentially expressed genes that were similarly 
expressed in both our \emph{in vivo} and \emph{in vitro} data sets.

\begin{figure}[ht]
\centering
\includegraphics{ini/supplement/commongenes.pdf}
\caption{Similarly expressed genes \emph{in vitro} and \emph{in vivo}}
\label{ini:fig:commons}
\end{figure}


\section{Discussion}

This is the first study to characterize the genome-wide transcriptional 
response to TcdA and TcdB in vivo. Additionally, several other parallel 
measurements were assessed to quantify changes at the cellular and 
tissue levels. The overall dynamics of the host responses to TcdA 
include rapid changes in cecal pathology, neutrophil infiltration, 
and gene expression. Conversely, TcdB elicits a delayed 
transcriptional response and causes significantly less pathology 
yet still recruits neutrophils and induces histopathological 
changes by 16h. The combined effects of TcdA and TcdB 
(20 \textmugreek{}g/toxin) on histopathology at 2h and the overall 
transcriptional response at 2h and 6h are not additive. 
However, two of three mice injected with TcdA+B did not 
survive to our 6h time point, so we do not rule out potential 
synergism at later times. For example, Hirota et al., using 
lower doses (5 \textmugreek{}g/toxin), found that TcdA and TcdB may act 
synergistically 4h after both toxins are introduced intrarectally 
\cite{Hirota:2012gx}. As for TcdB alone, several studies have 
found that TcdB does not damage hamster or mouse intestines, nor 
does TcdB bind to hamster brush border membranes 
\cite{Lyerly:1985dx,Rolfe:1991vx}. However, Lyerly et al. 
showed that, when the cecum was bruised before intragastric 
administration of TcdB, all mice became ill \cite{Lyerly:1985dx}. 
Hence, it is possible that the experimental procedure of cecal 
injection may allow or enhance the TcdB-induced pathology we 
observe at 16h. In line with our findings that TcdB has pathologic 
effects, Libby et al. found that 16 of 16 hamsters died within 36h 
of cecal injection of 60 \textmugreek{}g of TcdB and found that 35 \textmugreek{}g of TcdA 
resulted in epithelial lesions, edema, and neutrophil 
infiltration \cite{Libby:1982wm}. We were able to quantify separate 
aspects of these host responses over time, revealing the 
relative responses to TcdA and TcdB, individually altered 
genes and markers of intoxication, and regulated gene sets 
associated with pathways that function at the intracellular 
and extracellular levels.

This study builds on our previous analysis of the transcriptional 
response of a human ileocecal, epithelial cell line (HCT8) to 
TcdA or TcdB (2, 6, and 24h after toxin treatment). After mapping 
all orthologous genes, we found that the overall transcriptional 
response of HCT8 cells is poorly correlated to the responses of 
the cecal epithelial layer cells in vivo ($\text{r}^{2}<0.03$ 
for all comparisons, \autoref{ini:fig:correlations}). Some of 
these dissimilarities are presumably due to the different 
experimental systems and mRNA sources (in vitro vs. in vivo 
and human vs. mouse). Many differences may also represent 
important responses primarily observed in an in vivo 
experimental system. For instance, transient expression 
of chemokines and increased expression of several other 
cytokines was not observed in HCT8 cells. Also, the altered 
expression of many metabolic genes did not occur in HCT8 
cells. Conversely, cell-cycle and DNA damage-associated 
gene sets were not enriched in vivo as they were with HCT8 
cells. Selecting for expression that is similar between the 
data sets, ten genes are commonly upregulated (Rhob, Klf2, 
Klf6, Jun, Dusp1, Gdf15, Hspa1a, Dusp1, Bcl2l15, and Gpx2) and 
a few are commonly downregulated (Edn1, Alpi, and Bmp2; 
\autoref{ini:fig:commons}). In another high-throughput 
analysis of the host cell response to TcdA, Zeiser et al. 
analyzed the changes in the proteome of Caco2 cells 24h after 
TcdA exposure \cite{Zeiser:2013cu}. By mapping transcripts to 
proteins, we found that the proteomics data was poorly correlated 
with transcriptional changes in HCT8 cells and our in vivo 
data ($\text{r}^{2}<0.01$ for all comparisons). However, similar 
to our previous study and this current study, Zeiser et al. did 
note changes in the amount of many cell-cycle associated 
proteins and several proteins involved in lipid and cholesterol 
metabolism.

Aside from the quantitative and temporal differences of the 
physiological responses, many transcriptional similarities exist 
between the host response to TcdA and TcdB. For both toxins, the 
immediate transcriptional response, indicative of initial or 
acute toxin effects, is represented by altered expression of a 
small set of genes. By 6h, the number of differentially expressed 
genes for both toxins increases ~200 fold, coinciding with 
changes in pathology including neutrophil infiltration. TcdA 
challenge leads to approximately ten-fold more differentially 
expressed genes at 2, 6, and 16h; however, there is significant 
overlap in the genes affected by TcdA and TcdB (\autoref{ini:fig2p2}). 
Though TcdA-induced changes were greater in magnitude, correlation 
coefficients (which are scale invariant) between TcdA and TcdB 
demonstrate strong overall similarity in gene expression 
signatures (\autoref{ini:fig2p2}B). This difference in scale 
between the toxin responses may result from differences in molecular 
functions and/or the number, type, and sensitivity of cells affected. 
Which and how many cells are affected might also originate from the 
differential abilities of TcdA and TcdB to bind and enter 
intestinal cells. In line with our results showing that TcdB caused 
significantly less pathology than TcdA, Rolfe found little to no 
TcdB adsorption relative to TcdA on hamster brush border membranes 
\cite{Rolfe:1991vx}. The location and transport of toxins within 
the gut, which is very poorly understood, may also partly explain 
the extent of pathology in intoxicated mice. After cecal injection 
of TcdA or TcdB, we did not observe any significant pathology in 
the colon. Hence, the effects of the toxins which we measured were 
restricted to the cecum. Nevertheless, other systemic effects 
emanating from local insult may contribute to overall pathology. 
Multiple animal studies have observed increased mucosal permeability 
after TcdA intoxication, and Steele et al. demonstrated systemic 
dissemination of both TcdA and TcdB during severe infection in mice 
and piglets \cite{Steele:2012ft}. Despite the various explanations 
for the lesser effects we observe with TcdB, a change in 
transcription after injection of TcdB is distinct even at 2h and 
6h when changes in histopathology and many other variables are 
not apparent. Furthermore, this transcriptional response is 
highly correlated with the response to TcdA. Hence, the transcriptional 
analysis reveals that overall intracellular responses of 
epithelial-layer cells to TcdA and TcdB are largely similar though 
the magnitudes of the gross observed pathologies may differ.
 
Beyond piecemeal identification of genes with altered expression, 
the transcriptional data as a whole reflects the cellular responses 
to underlying molecular interactions. Our analyses identified the 
upregulation of several genes encoding Rho binding proteins and 
small GTPases that are known to be affected by TcdA and TcdB. We 
also identified strong upregulation of many interferon-inducible 
GTPases. These interferon-inducible GTPases have been implicated in 
several mechanisms of cell-autonomous immunity such as inflammasome 
activation, recognition of pathogens in vacuoles, assembly of 
defense complexes, and autophagy \cite{Kim:2012cu}. Though the 
transcription of interferon genes is unaltered in the 
epithelial-layer cells we isolated, the GTPase upregulation 
suggests the functional presence of interferons. Consistent 
with this, Ishida et al. found increased transcription of 
IFN-$\gamma$ in whole tissue and increased production of 
IFN-$\gamma$ by infiltrating neutrophils after injection of 
TcdA into ligated ileal loops of mice \cite{Ishida:2004ta}. 
In the same analysis, ``beta-tubulin binding'' and ``histone 
deacytelase activity'' were also enriched. Though the relevance 
of these associations may seem at first unclear, Nam et al. 
have recently shown that inhibition of histone deacytelase 6 
blocks TcdA-induced tubulin acetylation and subsequent 
mucosal damage \cite{Nam:2010er}. Hence, our analysis 
corroborates previous data and suggests that our transcriptional 
data reflects other functions that are affected in parallel 
with or prior to transcription. For example, a novel set of 
toxin-induced genes identified in our analysis includes 
several metabolic genes. At 2h, multiple genes associated 
with cholesterol and steroid synthesis (specifically the 
mevalonate pathway) are slightly upregulated, whereas, at 
6 and 16h, genes associated with fatty acid metabolism and 
detoxification of xenobiotics are downregulated. The 
expression of several of these genes is known to be controlled 
directly or indirectly by nuclear receptors. In light of the 
finding that gp96 (a paralogue to heat shock protein 90, Hsp90) 
serves as a receptor for TcdA in vitro, these transcriptional 
changes may also result from Hsp90 interactions (e.g. Hsp90 proteins 
bind xenobiotic response elements and steroid hormone 
receptors) \cite{Na:2008eu}.

In addition to the above intracellular effects, several 
genes previously associated with toxin-mediated pathophysiology 
are altered. For instance, our self-contained enrichment test 
identified the ``interleukin-1 mediated signaling pathway'' as 
the most enriched gene set 2h after TcdA injection. More specifically, 
TcdA increases proinflammatory Il1b expression by 80\% at 2h 
and then over 500\% at 16h; TcdB causes no such change. In 
addition to Il1b, concomitant increases in the IL-1 receptor 
antagonist, Il1rn, suggest a natural feedback mechanism. These 
findings are in line with a previous study demonstrating that 
recombinant IL1RN pretreatment attenuated TcdA+B-induced 
inflammation \cite{Ng:2010hu}. Expression of Il33, an IL-1 family 
cytokine involved in mucosal signaling, also dramatically increases 
in response to TcdA (28- and 95-fold at 6 and 16h, respectively). 
Similarly, in mice infected with \textit{C. difficile} (VPI10463), 
intestinal levels of IL-33 increase at the peak of infection (data 
not shown), suggesting that this cytokine is responsive in part to 
toxin effects. In another study, Hirota et al. measured cytokine 
levels in colonic tissue lysates four hours after intrarectal 
instillation of TcdA+TcdB. The data published by Hirota et al. 
are correlated to our measured cecal, epithelial-layer gene 
expression (r=0.5, TcdA+B at 6h) and intracellular protein 
expression (r=0.36, TcdA at 6h, \autoref{ini:fig:geneprotein}). 
Six hours after cecal injection of TcdA, serum cytokine 
concentrations correlate even more strongly with Hirota et 
al.'s data (r=0.68). Hence, our data from cecal cells and 
serum after cecal injection of TcdA and/or TcdB are in 
agreement with findings from colonic tissue after intrarectal 
instillation of TcdA+TcdB. Additionally, our data show the 
expression changes of many other genes over a 16h time course 
for each toxin individually. 

Although the expression and/or release of chemokines and cytokines 
are known responses to \textit{C. difficile} toxins, the full profile 
of the transcription of cytokines in response to toxins has not 
been investigated. Previously, the role of many chemokines 
and proinflammatory mediators in TcdA-induced enteritis has 
been studied individually with ileal loops or in vitro \cite{Sun:2010kt}. 
For example, Morteau et al. found increased Ccl3 and Ccl5 
transcription in whole tissue one hour after TcdA injection, 
and showed that Ccl3 knockout mice were less susceptible 
to TcdA \cite{Morteau:2002ts}. Castagliuolo et al. 
identified increased expression of Cxcl2 in rat epithelial 
cells after injection of TcdA into ligated ileal loops 
\cite{Castagliuolo:1998um}. Our results identify additional 
chemokines which are transiently expressed and provide 
insight into the response of the epithelial layer over 
a time course covering the development of toxin-mediated 
pathogenesis. Additionally, our data includes the 
expression of all cytokines in response to TcdA and TcdB 
separately and in combination. We found that overall 
cytokine expression in response to TcdA and TcdB is 
correlated, yet TcdA-induced changes are more pronounced. 
However, TcdB does not increase the expression of several 
inflammation-associated chemokines as TcdA does. This 
difference between toxins is particularly interesting 
in light of the fact that TcdB cecal injection causes 
less tissue damage and inflammation than injection of 
TcdA. Our data also show that the chemokine expression 
in response to TcdA is transient, yet many other cytokines 
continue to be expressed even after chemokine expression 
returns towards basal levels. The early chemokine 
expression may be involved in the acute toxin effects. 
However, since \textit{C. difficile} infection typically 
last several days, the continued expression of several other 
cytokines is potentially interesting and an unexplored area 
of research.

In order to characterize the timing and the effects of the 
acute response after toxin exposure, we investigated the 
early effects of CXCL1 and CXCL2 in the same cecal injection 
system. The increased expression of chemokine genes involved 
in neutrophil recruitment (e.g. Cxcl1 and Cxcl2) was 
coincident with neutrophil infiltration. Local epithelial 
damage and neutrophil infiltration were not attenuated by 
neutralization of CXCL1 and CXCL2. Similar findings in 
pathology between neutralizing antibodies and isotype controls 
could be due to insufficient levels of antibody near the 
site of toxin injection or the method of antibody administration. 
In another neutralization study, Castagliuolo et al. injected 
anti-CXCL2 intravenously into fasted rats 15 minutes prior to 
injection of TcdA into ileal loops; this neutralization attenuated 
TcdA-induced fluid secretion, mucosal permeability, and MPO 
activity \cite{Castagliuolo:1998um}. Other ileal loop experiments 
with TcdA have revealed that pathology and neutrophil infiltration 
become evident one to three hours after intoxication 
\cite{Ishida:2004ta,Castagliuolo:1998um}. Hence, it is unlikely the 
effects we measured in this study would have been discernible prior 
to our earliest 2h time point. On the other hand, few studies have 
examined the host responses after the initial acute response; most 
have focused on responses within the first 6 hours 
\cite{Morteau:2002ts,Kelly:1994cu,Kokkotou:2009ik,Ishida:2004ta,Alcantara:2005dt,Warny:2000ct,Castagliuolo:1994ta}. 
Our latest time point, 16h, did reveal attenuation of the 
TcdA-induced increase in systemic neutrophils for anti-CXCL1 
and anti-CXCL2-treated animals. Moreover, serum levels of 
CXCL1 and CXCL2 were predictors for survival. This result also 
suggests that surviving mice are poor responders in terms of Cxcl1 
and Cxcl2 expression and production. Related to this, a recent 
study by Feghaly et al. showed that inflammatory markers in stool, 
not the number of \textit{C. difficile} colony forming units, 
correlated with clinical outcomes \cite{ElFeghaly:2013gq}. In a 
hamster infection model, Steele et al. showed that serum levels of 
CXCL1, IL6, TNFÎ±, and IL1Î² are increased in cases of severe, 
systemic infection \cite{Steele:2012ft}. Our results demonstrating 
that high CXCL1 and CXCL2 correlate with mortality are supportive 
of these infection studies and suggest that the release of these 
and perhaps other inflammatory markers in serum is primarily 
toxin-mediated. Our experiments with anti-CXCL1 and anti-CXCL2 
thus emphasize the importance of the locale of toxin effects 
and chemokine expression systemically and/or throughout the intestine.

The multilevel measurements and our analysis have revealed 
important aspects of the relative host responses to TcdA versus 
TcdB intoxication, novel changes in transcription underlying 
observed physiological changes, and many aspects of the early 
time course and dynamics of the host response near the site of 
injection and in circulating blood. These extensive data and 
experimental framework also provide a basis for comparisons and 
future investigations with mutant toxins and mutant mouse strains. 
Furthermore, these data may be used to identify diagnostic markers 
or novel targets to attenuate host responses to TcdA and TcdB. 

\section{Acknowledgements}
This chapter was taken from \fullcite{DAuria:2013jo}.
I thank the co-primary author Glynis Kolling and all other co-authors.
We thank David Bolick for his technical expertise; David Lyerly 
at TECHLAB, Inc., for providing purified TcdA and TcdB; and Stephen 
Becker and Dr. Eric Houpt for providing immortalized mouse epithelial cells.


